### ISSUE TITLE
[Feature Request] Allow Apktool to run on device

### ISSUE THREAD

**Issue by thejunkjon** [iBotPeaches/Apktool#2034]
Hi, I am in the process of trying to get Apktool to run directly on the device itself.  From my understanding, the biggest issue in doing this is that Apktool relies on some of the SDK tools (aapt and aapt2) which, at the moment, are only built for non-ARM devices (https://github.com/iBotPeaches/Apktool/tree/master/brut.apktool/apktool-lib/src/main/resources/prebuilt).

I am currently attempting to build these tools for ARM (https://github.com/thejunkjon/android-tools).  Once this is done, would there be any other reason why Apktool wouldn't work directly on the device?  Thanks!

Edit: Please feel free to close this issue if this is not the appropriate channel to ask these type of questions.

**Comment by iBotPeaches** [comment]
> would there be any other reason why Apktool wouldn't work directly on the device? 

With modern Android supporting some of the Java 7 features we use, it should work. The work for ARM aapt was already done and rejected as a feature by myself here - https://github.com/iBotPeaches/Apktool/pull/1878#issuecomment-448373536

It is simply too much work to make official and mainline into Apktool.

**Comment by thejunkjon** [comment]
No worries. Thanks @iBotPeaches for the consideration.
